bin/i18n/resources/animations.rb

```diff
@@ -0,0 +1,27 @@
+require 'fileutils'
+require 'json'
+
+require_relative '../../animation_assets/manifest_builder'
+require_relative '../i18n_script_utils'
+
+module I18n
+  module Resources
+    module Animations
+      CROWDIN_FILE_PATH = "#{I18N_SOURCE_DIR}/animations/spritelab_animation_library.json".freeze
```

Comment:
To me `SOURCE_FILE_PATH` sounds more like a place from where we get the data rather than where we put it.
@Hamms @mgc1194 what do you think? How about `OUTCOME_FILE_PATH` or `RESULT_FILE_PATH` instead?

Reply:
Or, since we going to define two methods `sync_in` and `sync_out` in the module, we should name it more specifically something like `SYNC_IN_RESULT_FILE_PATH`

Reply:
In the context of the i18n sync, "the source" directory" is what we call the directory that is treated as the source of truth for i18n strings, and the purpose of the sync in is to populate the source directory with those strings in the appropriate format

Reply:
Note: https://github.com/code-dot-org/code-dot-org/blob/c3058c66901d5f86f65e6c8e572761ecb708f4ef/bin/i18n/i18n_script_utils.rb#L9